docs/storage/storage.md

```diff
@@ -0,0 +1,169 @@
+# AngularFireStorage
+
+> Cloud Storage is designed to help you quickly and easily store and serve user-generated content, such as photos and videos.
+
+### Import the NgModule
+
+Cloud Storage for AngularFire is contained in the `angularfire2/storage` module namespace. Import the `AngularFireStorageModule` in your `NgModule`. This sets up the `AngularFireStorage` service for dependency injection.
+
+```ts
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { AppComponent } from './app.component';
+import { AngularFireModule } from 'angularfire2';
+import { AngularFireStorageModule } from 'angularfire2/storage';
+import { environment } from '../environments/environment';
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    AngularFireModule.initializeApp(environment.firebase),
+    AngularFireStorageModule
+  ],
+  declarations: [ AppComponent ],
+  bootstrap: [ AppComponent ]
+})
+export class AppModule {}
+```
+
+### Injecting the AngularFireStorage service
+
+Once the `AngularFireStorageModule` is registered you can inject the `AngularFireStorage` service.
+
+```ts
+import { Component } from '@angular/core';
+import { AngularFireStorage } from 'angularfire2/storage';
+
+@Component({
+  selector: 'app-component',
+  template: ``
+})
+export class AppComponent {
+  constructor(private afs: AngularFireStorage) { }
+}
+```
+
+### Uploading blobs
+
+There are two options for uploading files. 
+
+| method   |                    | 
+| ---------|--------------------| 
+| `put(data: Blob, metadata?: storage.UploadMetadata | undefined): AngularFireUploadTask` | Starts the upload of the blob to the storage reference's path. Returns an `AngularFireUploadTask` for upload monitoring. | 
+| `putString(keyRefOrSnap: string, value: T)` | Firebase | AFUnwrappedSnapshot, value: Object) | Updates an existing item in the array. Accepts a key, database reference, or an unwrapped snapshot. |
+
+```ts
+import { Component } from '@angular/core';
+import { AngularFireStorage } from 'angularfire2/storage';
+
+@Component({
+  selector: 'app-root',
+  template: `
+  <input type="file" (onchange)="uploadFile($event)">
+  `
+})
+export class AppComponent {
+  constructor(private afs: AngularFireStorage) { }
+  uploadFile(event) {
+    const file = event.target.files[0];
+    const filePath = 'name-your-file-path-here';
+    const task = this.storage.upload(filePath, file);
```

Comment:
storage undefined.
Change `private afs: AngularFireStorage` to `private storage: AngularFireStorage`.

Reply:
Thanks for the catch!